README.rst

```diff
@@ -990,6 +992,106 @@ so they can be overridden if needed.
 In the example above, the "SSID" template is flagged as "(required)"
 and its checkbox is always checked and disabled.
 
+How to configure tunnels using WireGuard
```

Comment:
we should also explain VXLAN over WireGuard:

- we should explain that this mode configures layer 2 VXLAN tunnels encapsulated in WireGuard tunnels (layer 3)
- we can either explain it in the same section or create a new section that just explains the differences with setting up a pure wireguard tunnel